backend/lambdas/tasks/generate_queries.py

```diff
@@ -15,115 +16,111 @@
 queue = sqs.Queue(os.getenv("QueryQueue"))
 jobs_table = ddb.Table(os.getenv("JobTable", "S3F2_Jobs"))
 data_mapper_table_name = os.getenv("DataMapperTable", "S3F2_DataMappers")
-
-ARRAYSTRUCT = "array<struct>"
-ARRAYSTRUCT_PREFIX = "array<struct<"
-ARRAYSTRUCT_SUFFIX = ">>"
-STRUCT = "struct"
-STRUCT_PREFIX = "struct<"
-STRUCT_SUFFIX = ">"
-SCHEMA_INVALID = "Column schema is not valid"
-ALLOWED_TYPES = [
-    "bigint",
-    "char",
-    "double",
-    "float",
-    "int",
-    "smallint",
-    "string",
-    "tinyint",
-    "varchar",
-]
```

Comment:
This stuff (and more at the bottom of the file) is all needed when using column identifiers nested inside structs - I guess you removed it because you didn't need it, or was it just a merge conflict?